docs/cep/query-guide/query.md

```diff
@@ -376,7 +376,7 @@ Query to find out the maximum temperature out of the events arrived **during las
 ```
 insert into MaxTempStream
 select max(temp) as maxTemp
-from TempStream#window.time(10 min);
+from TempStream window sliding_time(10 min);
 ```
 
 Here, the `time` window operates in a sliding manner with millisecond accuracy, where it will process events in the following 3 time durations and output aggregated events when a list of events are received in a sequential order.
```

Comment:
```suggestion
Here, the `time` window operates in a sliding manner with millisecond accuracy, where it will process events in the following three time durations and output aggregated events when a list of events are received in a sequential order.
```